src/VisualStudio/Core/Def/Implementation/Options/LocalUserRegistryOptionPersister.cs

```diff
@@ -154,6 +155,19 @@ bool IOptionPersister.TryPersist(OptionKey optionKey, object value)
                         subKey.SetValue(key, value, RegistryValueKind.QWord);
                         return true;
                     }
+                    else if (optionKey.Option.Type.IsEnum)
```

Comment:
can we just define the value as int? rather than Enum? well, Enum might be fine. just saying since I think I saw some issue here before.

Reply:
There was a problem with Enums, which is why I added support for enums here.

Reply:
sure. if you are adding support to Enum, it would be nice to add the support to all of OptionPersister rather than just this one. since once people start to use Enum option, they won't know which specific OptionPersister support and which one doesn't

Reply:
Looks like the RoamingVisualStudioProfileOptionPersister already has enum support: http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/Options/RoamingVisualStudioProfileOptionPersister.cs,107. The other two implementions of IOptionPersister aren't relevant for this.